FacebookStrategy.php

```diff
@@ -63,7 +63,12 @@ public function int_callback(){
 			parse_str($response, $results);
 
 			if (!empty($results) && !empty($results['access_token'])){
-				$me = $this->me($results['access_token']);
+				
+				$fields = array();
```

Comment:
@jsenin I was working on the same issue, spotted yours when preparing a PR :)
my solution was to query for all the fields which are used then in int_callback() method (email, first_name, last_name, location, link and website), this wouldn't require any development for existing applications and attempting to retrieve a field which an app doesn't have access to doesn't trigger any errors